stackanetes/common/utils.py

```diff
@@ -21,3 +25,35 @@ def env(*args, **kwargs):
         if value:
             return value
     return kwargs.get('default', '')
+
+
+def get_kubectl_command(add_namespace=True):
```

Comment:
I don't wanna enforce. If operator doesn't fill namesapce in configuration file namespace will be set as "default".